crates/turborepo-lockfiles/src/pnpm/data.rs

```diff
@@ -80,7 +80,8 @@ pub struct Dependency {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct PackageSnapshot {
-    // can we make this flow?/is it necessary?
+    // We don't use any fields in resolution in order to avoid losing data we parse it as a generic
```

Comment:
I'm a little confused by this sentence. Is it:

> We don't use any fields in resolution; in order to avoid losing data we parse it as a generic

Or

> We don't use any fields in resolution in order to avoid losing data. We parse it as a generic

Reply:
Oops, that was an outdated comment. Removing it.